relying-party/src/main.jsx

```diff
@@ -8,9 +8,11 @@ import '@fontsource/roboto/700.css';
 import './main.css'
 
 import MainLayout from './layouts/MainLayout.jsx';
+import Results from './components/Results.jsx';
 
 ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
     <MainLayout/>
```

Comment:
This \<MainLayout /> component should surround any components intended to be the content of the page.

For example:
 - \<MainLayout>\<Results />\</MainLayout>